src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -36,74 +38,92 @@ const propTypes = {
 class ACHContractStep extends React.Component {
     constructor(props) {
         super(props);
+        this.validate = this.validate.bind(this);
 
         this.addBeneficialOwner = this.addBeneficialOwner.bind(this);
         this.submit = this.submit.bind(this);
 
         this.state = {
             ownsMoreThan25Percent: ReimbursementAccountUtils.getDefaultStateForField(props, 'ownsMoreThan25Percent', false),
             hasOtherBeneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'hasOtherBeneficialOwners', false),
-            acceptTermsAndConditions: ReimbursementAccountUtils.getDefaultStateForField(props, 'acceptTermsAndConditions', false),
-            certifyTrueInformation: ReimbursementAccountUtils.getDefaultStateForField(props, 'certifyTrueInformation', false),
             beneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'beneficialOwners', []),
```

Comment:
**question:** Why do we keep `ownsMoreThan25Percent`, `hasOtherBeneficialOwners` and `beneficialOwners` in `state` while the rest of the fields come from `props`?

Reply:
Can't we still dynamically render the forms based on the values in `props`? Why do they have to be duplicated in `state`?

Reply:
Oh good point, at least for `ownsMoreThan25Percent`/`hasOtherBeneficialOwners` I think those values are stored in reimbursementAccountDraft. Would it be better to remove state for those two and replace all instances like 
``` jsx
this.state.hasOtherBeneficialOwners
```
with
```jsx
this.props.reimbursementAccountDraft.hasOtherBeneficialOwners
```

Reply:
Seems like the beneficial owners too could be stored via props? My thinking atm is something like:

1. Feels like we have got the data structure for the beneficial owners wrong and should fix that up somehow
2. `reimbursementAccountDraft` as a concept feels no longer needed. The `Form` component manages drafts automatically and one of the main features it has built in:

https://github.com/Expensify/App/blob/7f276b6d02a6e596abe314ff44a6197b8f2397d4/src/components/Form.js#L237-L239